blockchain/blockchain-wallet-v4-frontend
# feat: BIND integration, allow Argentinian users to make deposits in ARS

## Description
Allow users from Argentina to make deposits in the wallet to buy crypto with ARS currency. Also we split the code of the bank details to allow developers to add another new currency more easily.

## Testing Steps
1. Create an Argentinian (AR) user.
2. In backoffice:
  - Request document (only one is enough)
  - Update the user document using this one 20312528046
  - Update the user address
  - Approved KYC
  - Verifiy tier 2
3. Login inside the wallet.
4. Select the ARS currency.
5. Make a deposit in ARS.
6. And you will see the bank transfer details for Argentinians.

<img width="1228" alt="Screen Shot 2022-05-23 at 15 33 04" src="https://user-images.githubusercontent.com/97447855/169884319-b755f615-71d3-418f-a40e-8fe2fee62d9d.png">
